firebase/rules/tests/executions.spec.ts

```diff
@@ -42,6 +54,59 @@ describe('/executions', () => {
     targaryen.util.setFirebaseRules(rules);
   });
 
+  it('Only system-user can write into common executions', () => {
+    expect(anonymousUser).cannotWrite('/executions/1/common', testExecution);
+    expect(currentUser).cannotWrite('/executions/1/common', testExecution);
+    expect(systemUser).canWrite('/executions/1/common', testExecution);
+  });
```

Comment:
According to database.rules

```
path /executions/{id}/common is Execution {
  read() { validateReadExecutionPrivacy(prior(this)) }
  write() { isOwningUser(this, 'user_id') || isSystem() }
}
```

Owners of an execution can write into `executions/common` as well. That's because owners can for example update the name of the execution. So this test should be updated accordingly.

Reply:
Got it, thanks for clarifying!